### ISSUE TITLE
Landing page like heimdall

### ISSUE THREAD

**Issue by softwarecrash** [ioBroker/ioBroker.web#241]
hello,
a nice to have....
the start page from web adapter show a admin, floot link and someelse, so a option to add other links.

the idea is only a result in a try to use heimdall(docker) and the proxy adapter to redirect from the web landing page to heimdall, but that doesnt work in any way

**Comment by Apollon77** [comment]
Did you ever tries to call the web port diretcly? ... try it ... an automatically generated page is already there ... all adapters that are set up for this web instance are shown there.

what else you need?

**Comment by softwarecrash** [comment]
> Did you ever tries to call the web port diretcly? ... try it ... an automatically generated page is already there ... all adapters that are set up for this web instance are shown there.
> 
> what else you need?

yes this page i mean, a way to self add url buttons to this page

**Comment by o0shojo0o** [comment]
You can do that:
![image](https://user-images.githubusercontent.com/19333515/199080843-7680c22c-7824-440d-881d-4f289927aad9.png)
![image](https://user-images.githubusercontent.com/19333515/199080866-ae44b5fc-8500-431e-993e-81a846f199fa.png)
